composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -152,13 +153,20 @@ export default function (
     }
   })
 
+  const actionsId = computed(() =>
+    actions.value.map(({ action }) => JSON.stringify(action)).join('_'),
+  )
+
   watch(
-    actions,
+    actionsId,
     async () => {
       if (fees.actionAutoFees) {
         try {
           const feesPromisses = actions.value.map(
             async ({ action, prefix }) => {
+              if (!isActionValid(action)) {
+                return '0'
```

Comment:
Should we throw an error or toast error here, instead of returning the empty value?

Reply:
i'm not sure.
@exezbcz

Reply:
I see so we may not need the warning toast as it's a normal path.